<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ItemEditorDialog</class>
 <widget class="QDialog" name="ItemEditorDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>490</width>
    <height>250</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ChangeMe!</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <property name="spacing">
    <number>4</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="lblCode">
     <property name="text">
      <string>&amp;Код</string>
     </property>
     <property name="buddy">
      <cstring>edtCode</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="edtCode"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="lblRegionalCode">
     <property name="text">
      <string>&amp;Региональный код отношения</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalCode</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="3">
    <widget class="QLineEdit" name="edtRegionalCode"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="lblLeftName">
     <property name="text">
      <string>&amp;Субьект отношения</string>
     </property>
     <property name="buddy">
      <cstring>edtLeftName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="edtLeftName">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(лат.) предметы, всякое лицо, вещь, о коих говорится.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Словарь Даля &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLabel" name="lblRightName">
     <property name="text">
      <string>&amp;Объект отношения</string>
     </property>
     <property name="buddy">
      <cstring>edtRightName</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QLineEdit" name="edtRightName">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;(лат.) предмет противоположный субъекту.&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;--&lt;/span&gt;&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Словарь Даля&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="lblGenetic">
     <property name="text">
      <string>&amp;Генетическая связь</string>
     </property>
     <property name="buddy">
      <cstring>chkDirectGeneticRelation</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="chkDirectGeneticRelation">
     <property name="toolTip">
      <string>Передача генетического материала от субъекта к объекту</string>
     </property>
     <property name="text">
      <string>Прямая</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QCheckBox" name="chkBackwardGeneticRelation">
     <property name="toolTip">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:8pt;&quot;&gt;Передача генетического материала от объекта к субъекту&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Обратная</string>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="lblRepresentative">
     <property name="text">
      <string>&amp;Представительская связь</string>
     </property>
     <property name="buddy">
      <cstring>chkDirectRepresentativeRelation</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QCheckBox" name="chkDirectRepresentativeRelation">
     <property name="toolTip">
      <string>Субъект является представителем объекта</string>
     </property>
     <property name="text">
      <string>Прямая</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QCheckBox" name="chkBackwardRepresentativeRelation">
     <property name="toolTip">
      <string>Объект является представителем субъекта</string>
     </property>
     <property name="text">
      <string>Обратная</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="lblEpidemic">
     <property name="text">
      <string>&amp;Эпид.контактирование</string>
     </property>
     <property name="buddy">
      <cstring>chkDirectEpidemicRelation</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="chkDirectEpidemicRelation">
     <property name="toolTip">
      <string>Возможна передача от субъекта к объекту</string>
     </property>
     <property name="text">
      <string>Прямое</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QCheckBox" name="chkBackwardEpidemicRelation">
     <property name="toolTip">
      <string>Возможна передача от объекта к субъекту</string>
     </property>
     <property name="text">
      <string>Обратное</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="lblDonation">
     <property name="text">
      <string>&amp;Донорство</string>
     </property>
     <property name="buddy">
      <cstring>chkDirectDonationRelation</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="chkDirectDonationRelation">
     <property name="toolTip">
      <string>Передача органов от субъекта к объекту</string>
     </property>
     <property name="text">
      <string>Прямое</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QCheckBox" name="chkBackwardDonationRelation">
     <property name="toolTip">
      <string>Передача органов от объекта к субъекту</string>
     </property>
     <property name="text">
      <string>Обратное</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="lblLeftSex">
     <property name="text">
      <string>Пол субъекта</string>
     </property>
     <property name="buddy">
      <cstring>cmbLeftSex</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QComboBox" name="cmbLeftSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>мужской</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>женский</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="lblRightSex">
     <property name="text">
      <string>Пол объекта</string>
     </property>
     <property name="buddy">
      <cstring>cmbRightSex</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="3">
    <widget class="QComboBox" name="cmbRightSex">
     <item>
      <property name="text">
       <string/>
      </property>
     </item>
     <item>
      <property name="text">
       <string>мужской</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>женский</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="9" column="0">
    <spacer>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>103</width>
       <height>16</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="3">
    <widget class="QLineEdit" name="edtRegionalReverseCode"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="lblRegionalReverseCode">
     <property name="text">
      <string>Региональный код &amp;обратного отношения</string>
     </property>
     <property name="buddy">
      <cstring>edtRegionalReverseCode</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>edtCode</tabstop>
  <tabstop>edtRegionalCode</tabstop>
  <tabstop>edtRegionalReverseCode</tabstop>
  <tabstop>edtLeftName</tabstop>
  <tabstop>edtRightName</tabstop>
  <tabstop>chkDirectGeneticRelation</tabstop>
  <tabstop>chkBackwardGeneticRelation</tabstop>
  <tabstop>chkDirectRepresentativeRelation</tabstop>
  <tabstop>chkBackwardRepresentativeRelation</tabstop>
  <tabstop>chkDirectEpidemicRelation</tabstop>
  <tabstop>chkBackwardEpidemicRelation</tabstop>
  <tabstop>chkDirectDonationRelation</tabstop>
  <tabstop>chkBackwardDonationRelation</tabstop>
  <tabstop>cmbLeftSex</tabstop>
  <tabstop>cmbRightSex</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>171</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>117</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ItemEditorDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>251</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>247</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
